snippets/variant-list-row.liquid

```diff
@@ -0,0 +1,233 @@
+    </div>
+  </td>
+
+  <td class="variant-item__details medium-hide large-up-hide">
+    <div class="variant-item__info">
+      <span class="variant-item__name h4 break">{{ item.title | escape }}</span>
+      {%- if section.settings.show_sku -%}
+        <span class="variant-item__sku caption break">{{ sku | escape }}</span>
+      {%- endif -%}
+    </div>
+
+    {%- render 'loading-overlay' -%}
+
+    {%- if item.compare_at_price != item.price -%}
+      <dl class="variant-item__discounted-prices">
+        <dt class="visually-hidden">
+          {{ 'products.product.price.regular_price' | t }}
+        </dt>
+        <dd>
+          <s class="variant-item__old-price price price--end">
+            {{ item.compare_at_price | money }}
+          </s>
+        </dd>
+        <dt class="visually-hidden">
+          {{ 'products.product.price.sale_price' | t }}
+        </dt>
+        <dd class="price">
+          {{ item.price | money -}}
+          <span>{{ 'sections.variant_list.each' | t }}</span>
+        </dd>
+      </dl>
+    {%- else -%}
+      <span class="price">
+        {{ item.price | money -}}
+        <span>{{ 'sections.variant_list.each' | t }}</span>
+      </span>
+    {%- endif -%}
+
+    {%- if item.available and item.unit_price_measurement -%}
+      <div class="unit-price caption">
+        <span class="visually-hidden">{{ 'products.product.price.unit_price' | t }}</span>
+        {{ item.unit_price | money }}
+        <span aria-hidden="true">/</span>
+        <span class="visually-hidden">&nbsp;{{ 'accessibility.unit_price_separator' | t }}&nbsp;</span>
+        {%- if item.unit_price_measurement.reference_value != 1 -%}
+          {{- item.unit_price_measurement.reference_value -}}
+        {%- endif -%}
+        {{ item.unit_price_measurement.reference_unit }}
+      </div>
+    {%- endif -%}
+  </td>
+
+  <td class="variant-item__quantity small-hide">
+    <div class="variant-item__quantity-wrapper">
+      <label class="visually-hidden" for="Quantity-{{ variant.id }}">
+        {{ 'products.product.quantity.label' | t }}
+      </label>
+      {%- liquid
+        assign check_against_inventory = true
+        if variant.inventory_management != 'shopify' or variant.inventory_policy == 'continue'
+          assign check_against_inventory = false
+        endif
+        if variant.quantity_rule.min > variant.inventory_quantity and check_against_inventory
+          assign quantity_rule_soldout = true
+        endif
+      -%}
+      <div class="variant-item__quantity-info">
+        {% render 'icon-info' %}
+      </div>
+      {%- if variant.available == false or quantity_rule_soldout -%}
+        <span class="variant-item__sold-out h4"> {{ 'products.product.sold_out' | t }} </span>
+      {%- else -%}
+        {% # theme-check-disable %}
+        {% assign cart_qty = cart | item_count_for_variant: variant.id %}
+        {% # theme-check-enable %}
+        {% render 'quantity-input', variant: variant %}
```

Comment:
When tabbing through these the focus outlines can appear above the totals
https://screenshot.click/09-04-g3d0b-xh638.mp4

Reply:
It's also a little annoying that the window won't automatically scroll when the focus gets to the border of the totals element, since that's effectively serving at the viewport boundary now. Maybe some scroll-padding could assist with that? Not sure how the sticky changes do or will affect these behaviors though.

Reply:
This is fixed in the sticky bar PR https://github.com/Shopify/dawn/pull/2690 

👀 you can see it works as expected:https://screenshot.click/09-03-itupl-a9o4e.mp4